apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -11,47 +11,113 @@
         </v-list-item-avatar>
-
         <v-list-item-content>
-          <v-list-item-title>Comparison</v-list-item-title>
+          <v-list-item-title>
+            Profile View
+          </v-list-item-title>
         </v-list-item-content>
       </v-list-item>
+      <v-subheader>Files</v-subheader>
+      <v-expansion-panels flat="true" v-model="file_views">
+        <v-expansion-panel>
+          <v-expansion-panel-header title="View controls' results">
+            <v-list-item>Results</v-list-item>
+          </v-expansion-panel-header>
+          <v-expansion-panel-content>
+            <div
+              v-if="
+                visible_evaluation_files != null &&
+                  visible_evaluation_files.length > 0
+              "
+            >
+              <FileItem
+                v-for="(file, i) in visible_evaluation_files"
+                :key="i"
+                :file="file"
+              />
+              <v-list-item
+                @click="toggle_all_evaluations"
+                title="Show all files' controls"
+              >
+                <v-list-item-avatar>
+                  <v-icon small>mdi-format-list-bulleted</v-icon>
+                </v-list-item-avatar>
+                <v-list-item-content>
+                  <v-list-item-title>
+                    <div v-if="all_toggled_evaluation">
+                      Deselect all results set
+                    </div>
+                    <div v-else>Select all results set</div>
+                  </v-list-item-title>
+                </v-list-item-content>
+              </v-list-item>
+            </div>
+          </v-expansion-panel-content>
+        </v-expansion-panel>
+        <v-expansion-panel>
+          <v-expansion-panel-header title="View controls' results">
+            <v-list-item>Profiles</v-list-item>
+          </v-expansion-panel-header>
+          <v-expansion-panel-content>
+            <div
+              v-if="
+                visible_profile_files != null &&
+                  visible_profile_files.length > 0
+              "
+            >
+              <ProfileItem
+                v-for="(file, i) in visible_profile_files"
+                :key="i"
+                :file="file"
+              />
+              <v-list-item
+                @click="toggle_all_profiles"
+                title="Show all files' controls"
+              >
+                <v-list-item-avatar>
+                  <v-icon small>mdi-format-list-bulleted</v-icon>
+                </v-list-item-avatar>
+                <v-list-item-content>
+                  <v-list-item-title>
+                    <div v-if="all_toggled_profiles">Deselect all profiles</div>
+                    <div v-else>Select all profiles</div>
+                  </v-list-item-title>
+                </v-list-item-content>
+              </v-list-item>
+            </div>
+          </v-expansion-panel-content>
+        </v-expansion-panel>
```

Comment:
The whole `v-expansion-panel` here should be its own component, and it should be reused for the Results and Profiles dropdowns.

Reply:
Moving this to be addressed when issue #82 is addressed.

[https://github.com/mitre/heimdall2/issues/82#issuecomment-691059364](url)